### ISSUE TITLE
Radar and Scatter Charts Give Errors When Opened in Excel 2013

### ISSUE THREAD

**Issue by ComputerTinker** [PHPOffice/PHPExcel#645]
In preparation for generating my own charts I am looking at the examples which are provided along with the source code. I ran each of the example files through PHPExcel in order to create the .xlsx files, and no errors were reported through the command line for either the "radar" or "scatter" charts:

![shot](https://cloud.githubusercontent.com/assets/9611807/9316921/d1de649e-44fd-11e5-93fb-bcc9e34ecd31.png)

I then opened each of .xlsx files and all worked fine except for the "radar" and "scatter" charts which give the following error:

![shot2](https://cloud.githubusercontent.com/assets/9611807/9316947/0de98a40-44fe-11e5-88dd-100a991ac090.png)

If I click "Yes" to the prompt, I can see the data in the workbook, but no chart exists.

**Comment by Jarvizx** [comment]
any solution?

**Comment by ComputerTinker** [comment]
No. Doesn't help that I have no real error messages to work off of. :-(

**Comment by sseffa** [comment]
+1

**Comment by Jarvizx** [comment]
nice, last update?

**Comment by sseffa** [comment]
no, it's working on 1.8.0 version
![screenshot_23](https://cloud.githubusercontent.com/assets/2960378/11447729/e7324684-9553-11e5-9506-edac8eec86d4.png)

**Comment by petehenshall** [comment]
See https://github.com/PHPOffice/PHPExcel/issues/956
